### PR TITLE
Rename staging_docs to docs

### PR DESCRIPTION
**Pull request by pedro-psb**
This does:

- Remove legacy docs
- Rename staging_docs to docs

[noissue]

### CODE REVIEW

File: staging_docs/user/guides/publish.svg
Comment (quba42) [inline staging_docs/user/guides/publish.svg:1]: Don't we still need the images for the new docs?

**Comment by quba42** [comment]
I don't really know how to review this. I presume it all makes sense?

Would it perhaps make sense to put the removal and the rename into two separate commits? Might make it easier to understand the two commits individually.

**Comment by mdellweg** [comment]
You can run `pulp-docs serve` or similar, when you want to preview docs from a local checkout.

File: staging_docs/user/guides/publish.svg
Comment (pedro-psb) [reply staging_docs/user/guides/publish.svg:1]: Yes, but the staging_docs did had copies of the images (those were just renamed).
I will make remove/rename into two commits so its easier to see what's happening.

**Review by quba42 — APPROVED** [review]
Ok, the two separate commits really helped me unblock my brain: This is just doing exactly what it says in the description: Remove the still present legacy docs, and then move the new docs to the now empty location. :laughing: I also see this has already happened for other plugins.

If I want to check something in the legacy docs I will just need to go back in the history to before this change. That all LGTM.

**Pull request merged**